### PR TITLE
Small changes to README.md

### PR DESCRIPTION
**Pull request by dandare100**
Thank you for this project, I think it's great.

Small changes I had to do to get it running.

Just sharing.

### CODE REVIEW

**Comment by evilsocket** [comment]
hi, and thanks! are you sure about the docker fix tho? i'm pretty sure that is the correct command line to create a container

File: README.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ commands:
 
 And:
 
-	docker container create --name medusajail nginx # feel free to pick any image
+	docker container run --name medusajail nginx # feel free to pick any image
```
Comment (evilsocket) [inline README.md]: the original command line is correct and it will create a container, please revert this change

Comment (dandare100) [reply README.md]: Hello, will do, I just want to double check with you please, 

It will create the container but not start it.

Here is the output of a telnet HP if create is used

```
[admin@TNAS ~]$ ls
Error response from daemon: Container 0c8590cd3ea557ac4d9dc112efb208cf3ce65daab682ba0dc6334631daeaaed4 is not running
[admin@TNAS ~]$ 
```

Comment (evilsocket) [reply README.md]: yes that is correct, you have to start it manually... i have just changed the README to reflect this information, does it make more sense now?

**Comment by dandare100** [comment]
No need to merge this dude, its to small.

I had no other way of letting you know.

I'll close it ,

**Comment by evilsocket** [comment]
thanks a lot, really appreciated
